modules/ocf_apphost/manifests/systemd_linger.pp

```diff
@@ -0,0 +1,6 @@
+# Declares that the following user is to have a persistent per-user systemd
+define ocf_apphost::systemd_linger($user = $title) {
+  file { "/var/lib/systemd/linger/${user}":
```

Comment:
The way this is enabling lingering is by manually sticking files in the `/var/lib/systemd/linger/` directory. This makes it straightforward for Puppet to disable lingering for users who are removed from ocfdev. But it's also some of an abstraction barrier violation, and I can't be certain it will work forever.

The other way to do it is by exec'ing `loginctl enable-linger ${user}` as the manual says. This is maybe more robust, but then it's not easy to disable lingering for users who get removed from ocfdev.

What do you think?